client/src/app/mock-ingredients.ts

```diff
@@ -0,0 +1,16 @@
+import { Ingredient } from './ingredient';
+
+export const INGREDIENTS: Ingredient[] = [
```

Comment:
This mock data will be provided by the `FridgeService` for the time being until backend API is ready.